pkgs/by-name/al/alsa-plugins/package.nix

```
@@ -13,7 +23,7 @@ stdenv.mkDerivation rec {
 
   # ToDo: a52, etc.?
   buildInputs =
-    [ alsa-lib libogg ]
+    [ alsa-lib libogg libsamplerate ffmpeg.dev ]
```

```suggestion
    [ alsa-lib libogg libsamplerate ffmpeg ]
```
the dev variant should be choosen automatically, isn't it?